common/xrdp_client_info.h

```diff
@@ -172,9 +183,12 @@ struct xrdp_client_info
 
     int enable_token_login;
     char domain_user_separator[16];
+
+    /* xrdp.overrode_* values */
```

Comment:
Small point. `xrdp.override_*` rather than `xrdp.overrode_*`.

Reply:
It was a typo. Sorry.
I will correct it.